sdk/download.go

```diff
@@ -269,11 +269,12 @@ func UpdateFile(file, url string, client *http.Client) error {
 	}
 	t.Close()
 	tempFile := t.Name()
-	defer os.Remove(tempFile)
+	os.Remove(tempFile) // let DownloadFile create it and use umask
```

Comment:
@marineam Sorry, I just pushed a `chmod` version and commented before your commend showed up.